### PR TITLE
Scripts code clean up & tweaks

### PR DESCRIPTION
**Pull request by FS-21**
Minor changes:
- Moved some Techno functions from ScriptExt to TechnoExt
- Tweaked a bit the existing script actions.
- Moved ApplyMindControlRangeLimit call into OnEarlyUpdate().

No Credits/What's new required since these changes aren't features.

### CODE REVIEW

**Comment by Starkku** [comment]
I'll do a more thorough review sometime in near future but is it really necessary to constantly check for null pointers everywhere?

File: src/Ext/Techno/Body.Update.cpp [hunk 1]
```diff
@@ -40,6 +40,7 @@ void TechnoExt::ExtData::OnEarlyUpdate()
 	this->ApplySpawnLimitRange();
 	this->UpdateLaserTrails();
 	this->DepletedAmmoActions();
+	ApplyMindControlRangeLimit(this->OwnerObject());
```
Comment (chaserli) [inline src/Ext/Techno/Body.Update.cpp:43]: no!

Comment (Metadorius) [reply src/Ext/Techno/Body.Update.cpp:43]: why?

Comment (chaserli) [reply src/Ext/Techno/Body.Update.cpp:43]: unrelated & unnecessary

**Pull request closed without merging**